lib/global-styles.php

```diff
@@ -171,22 +171,32 @@ function gutenberg_experimental_global_styles_get_theme() {
  * @return string CSS rule.
  */
 function gutenberg_experimental_global_styles_resolver( $global_styles ) {
-	$css_rule = '';
-
-	$token    = '--';
-	$prefix   = '--wp' . $token;
-	$css_vars = gutenberg_experimental_global_styles_get_css_vars( $global_styles, $prefix, $token );
-	if ( empty( $css_vars ) ) {
-		return $css_rule;
-	}
+	$css_rules = '';
+
+	// This is only for demo purposes, it should be pluggable
+	// so new selectors can be register
+	// (3rd party blocks, custom selectors, etc).
+	$selectors = array(
+		'core'                      => ':root',
+		'core/paragraph'            => '.wp-site-blocks p',
+		'custom/paragraph-in-quote' => '.wp-site-blocks .wp-block-quote p',
```

Comment:
I am not sure if blocks should be allowed to configure specific selectors, besides one that represents the block itself I guess on mobile, for example, knowing the values of custom elements like custom/paragraph-in-quote would be complex.
I think if blocks need multiple levels using InnerBlocks should be the solution. I guess we can discuss that later, but maybe for a first version, we can avoid this?

Reply:
@jorgefilipecosta I tend to agree. I think that list of configured selectors will be to get incredibly massive for themes that want to have more intricate designs. I would be curious to get your thoughts on the more agnostic approach I described here:
https://github.com/WordPress/gutenberg/issues/19611#issuecomment-603912611

Reply:
Hi @johnstonphilip, I was planning on proposing a PR that replaces current color classes with CSS variables and I think it will follow a path similar to the one described in https://github.com/WordPress/gutenberg/issues/19611#issuecomment-603912611.

Reply:
Removed this from the proposal.